tensorboard/webapp/notification_center/_redux/BUILD

```diff
@@ -0,0 +1,48 @@
+load("//tensorboard/defs:defs.bzl", "tf_ts_library")
+
+package(default_visibility = ["//tensorboard/webapp/notification_center:__subpackages__"])
+
+licenses(["notice"])
+
+tf_ts_library(
+    name = "_redux",
+    srcs = [
+        "notification_center_actions.ts",
+        "notification_center_effects.ts",
+        "notification_center_reducers.ts",
+        "notification_center_selectors.ts",
+    ],
+    deps = [
+        ":types",
+        "//tensorboard/webapp:app_state",
+        "//tensorboard/webapp/notification_center:types",
+        "@npm//@angular/core",
+        "@npm//@ngrx/effects",
+        "@npm//@ngrx/store",
+        "@npm//rxjs",
+    ],
+)
+
+tf_ts_library(
+    name = "types",
+    srcs = [
+        "notification_center_types.ts",
+    ],
+    deps = [
+        "//tensorboard/webapp/notification_center:types",
```

Comment:
I assumed the `notification_center/types.ts` to mostly re-export. This dependency direction feels odd.

Reply:
See the discussion above